projects/laji/src/app/+taxonomy/taxon/info-card/taxon-identification/taxon-identification.facade.ts

```diff
@@ -0,0 +1,113 @@
+import { Injectable, OnDestroy } from '@angular/core';
+import { BehaviorSubject, forkJoin, from, Subject, Subscription } from 'rxjs';
+import { map, distinctUntilChanged, switchMap, concatMap, scan, takeUntil } from 'rxjs/operators';
+import { Taxonomy } from 'projects/laji/src/app/shared/model/Taxonomy';
+import { Taxon } from '../../../../../../../laji-api-client/src/lib/models';
+import { TaxonomyApi } from 'projects/laji/src/app/shared/api/TaxonomyApi';
+import { TranslateService } from '@ngx-translate/core';
+import { TaxonomyModule } from '../../../taxonomy.module';
+
+interface IIdentificationState {
+  childTree: Taxonomy[];
```

Comment:
change this to `childTree?: Taxonomy[];` (or change the initial state to `childTree: []`)